cartridge/lua-api/compression.lua

```diff
@@ -0,0 +1,171 @@
+local lua_api_get_topology = require('cartridge.lua-api.get-topology')
+
+local log = require('log')
+local pool = require('cartridge.pool')
+local errors = require('errors')
+
+-- Surf by replicates to find master storages and call on them getStorageCompressionInfo()
+local function get_cluster_compression_info()
+    local replicasets, err = lua_api_get_topology.get_replicasets()
+    if replicasets == nil then
+        log.error(err)
+        return nil, err
+    end
+
+    local compression_info = {}
+
+    for _, rpl in pairs(replicasets or {}) do
+        for _, role in pairs(rpl.roles or {}) do
+            if role == 'vshard-storage' then
+                local master = rpl.master
+
+                local storage_compression_info, err = errors.netbox_call(
+                    pool.connect(master.uri, {wait_connected = true}),
+                    '_G.getStorageCompressionInfo', {master}, {timeout = 1}
+                )
+                if storage_compression_info == nil then
+                    error(err)
```

Comment:
Please follow the standard error handling style: the function should return `nil, err` instead of throwing the error directly. And don't forget to check errors when you call the function.